libs/wingsdk/.projenrc.ts

```diff
@@ -1,28 +1,6 @@
 import { readdirSync } from "fs";
 import { JsonFile, cdk, javascript } from "projen";
 
-const UNDOCUMENTED_CLOUD_FILES = ["index", "test-runner"];
-
-const cloudFiles = readdirSync("./src/cloud");
-
-const cloudResources: Set<string> = new Set(
-  cloudFiles.map((filename) => filename.split(".")[0])
-);
-
-UNDOCUMENTED_CLOUD_FILES.forEach((file) => cloudResources.delete(file));
-
-const undocumentedResources = Array.from(cloudResources).filter(
-  (file) => !cloudFiles.includes(`${file}.md`)
-);
-
-if (undocumentedResources.length) {
```

Comment:
I guess you moved this code because it didn't really work? Does it work now? IMO the build should fail when a new resource doesn't have docs